tests/go.mod

```diff
@@ -177,12 +178,12 @@ require (
 )
 
 replace (
+	cosmossdk.io/collections => ../collections
```

Comment:
Could these be grouped under `TODO tag all extracted modules after SDK refactor`